packages/shared/src/types/targeting/shared.ts

```diff
@@ -33,3 +34,12 @@ export interface PurchaseInfo {
     userType: purchaseInfoUser;
     product: purchaseInfoProduct;
 }
+
+export const abandonedBasketSchema = z.object({
```

Comment:
should this be exported somewhere else, eg how `epicPropsSchema` is in `dotcom/src/index.ts` ?

Reply:
I don't think that's necessary

Reply:
update for any future readers, yes it should. See #1140